.vscode/launch.json

```diff
@@ -7,94 +7,251 @@
-                "Identity-SelfHost",
+                "run-Admin-SelfHost",
+                "run-API-SelfHost",
+                "run-EventsProcessor-SelfHost",
+                "run-Identity-SelfHost",
             ],
             "presentation": {
                 "hidden": false,
                 "group": "AA_compounds",
                 "order": 99
             },
+            "preLaunchTask": "buildSelfHostOss",
             "stopAll": true
-        }
-    ],
-    "configurations": [
+        },
+        {
+            "name": "Admin",
+            "configurations": [
+                "run-Admin"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildAdmin",
+        },
+        {
+            "name": "API",
+            "configurations": [
+                "run-API"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildAPI",
+        },
+        {
+            "name": "Billing",
+            "configurations": [
+                "run-Billing"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildBilling",
+        },
+        {
+            "name": "Events Processor",
+            "configurations": [
+                "run-EventsProcessor"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildEventsProcessor",
+        },
+        {
+            "name": "Icons",
+            "configurations": [
+                "run-Icons"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildIcons",
+        },
         {
             "name": "Identity",
+            "configurations": [
+                "run-Identity"
+            ],
             "presentation": {
                 "hidden": false,
                 "group": "cloud",
-                "order": 10
+            },
+            "preLaunchTask": "buildIdentity",
+        },
+        {
+            "name": "Notifications",
+            "configurations": [
+                "run-Notifications"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildNotifications",
+        },
+        {
+            "name": "SSO",
+            "configurations": [
+                "run-Sso"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildSso",
+        },
+        {
+            "name": "Admin Self Host",
```

Comment:
❓ Is there anything VS provides to group self-host vs. not perhaps?

Reply:
These will be grouped in the UI due to the `presentation` field, but there is no grouping at the json level, no.